src/components/MainSection/tests/find.test.js

```diff
@@ -0,0 +1,97 @@
+import { find } from "../MainSection.utils";
+
+describe("Searching for strings function (ignoring case sensivity)", () => {
+  const input = {
```

Comment:
czasami też takie stałe wartości wyrzuca się do osobnego pliku, żeby testy zawierały tylko testy, ale to też nie jest zasada, różne są podejścia - piszę tylko informacyjnie :D